contract_sale_generation/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 
 {
     "name": "Contracts Management - Recurring Sales",
-    "version": "16.0.1.1.0",
+    "version": "17.0.1.1.0",
```

Comment:
@Boxnia Could you reset this to '17.0.1.0.0'?

Moreover, this kind of changes should be located in migration commit instead (not pre-commit).

Thanks